sos/report/plugins/cgroups.py

```diff
@@ -26,7 +26,8 @@ def setup(self):
 
         self.add_copy_spec([
             "/proc/cgroups",
-            "/sys/fs/cgroup"
+            "/sys/fs/cgroup",
+            "/proc/*/cgroup",
```

Comment:
I've tested this and it works, but this will go through all subdirs of /proc, and only collects 'cgroup' when it finds it. 
We do something similar but more elegantly in the 'process' plugin:

https://github.com/sosreport/sos/blob/96e4d79231819b45ef62629206a7a3da31b3a1dd/sos/report/plugins/process.py#L52-L58

So my question is, is my approach good enough, or should I use the one in the process plugin gathering the PID entries with:

`procs = [p for p in self.listdir("/proc") if re.match("[0-9]", p)]` 

Then iterating through them?

Reply:
`"/proc/[0-9]*/cgroup`